youtube_dl/extractor/megaphone.py

```diff
@@ -45,11 +68,225 @@ def _real_extract(self, url):
             'thumbnail': thumbnail,
             'title': title,
             'author': author,
-            'duration': episode_data['duration'],
+            'duration': episode_data.get('duration'),
             'formats': formats,
         }
 
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        episode_json = self._download_json(self._JSON_URL_TEMPL % video_id, video_id, fatal=False)
+        if episode_json is False:
```

Comment:
Here `if episode_json is False` can/should be replaced with `if not episode_json`, which is also the way you're doing it later.

Reply:
The test is intentional to distinguish between `False` (bad result from request) and a JSON object that converts to a falsy Python value (eg `{}`, `[]`).

Reply:
Oh, ok. That makes sense. Shoulda read the statement above it.